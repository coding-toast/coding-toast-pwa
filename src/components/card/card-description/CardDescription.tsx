import * as React from 'react'

interface CardDescriptionProps {
  text: string
}

const CardDesctiption: React.FC<CardDescriptionProps> = props => {
  const { text } = props

  return (
    <div>
      <p>{text}</p>
    </div>
  )
}

export default CardDesctiption
