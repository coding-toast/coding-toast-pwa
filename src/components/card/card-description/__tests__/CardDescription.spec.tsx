import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CardDescription from '../CardDescription'

describe('<CardDescription />', () => {
  it('renders correctly', () => {
    const component = shallow(<CardDescription text="Content" />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
