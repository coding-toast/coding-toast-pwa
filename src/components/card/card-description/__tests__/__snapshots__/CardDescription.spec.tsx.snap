// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardDescription /> renders correctly 1`] = `
<div>
  <p>
    Content
  </p>
</div>
`;
